pkg/cmd/stack/translate.go

```diff
@@ -57,6 +58,9 @@ const (
 
 	// oktetoComposeVolumeAffinityEnabledEnvVar represents whether the feature flag to enable volume affinity is enabled or not
 	oktetoComposeVolumeAffinityEnabledEnvVar = "OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED"
+
+	// dependsOnAnnotation represents the annotation to define the depends_on field
+	dependsOnAnnotation = "okteto.dev/depends-on"
```

Comment:
I'd follow the standard we have for labels/annotations. `dev.okteto.com/depends-on`